examples/va/dcn/README.md

```diff
@@ -0,0 +1,42 @@
+# Distributed Compute Node (DCN) OpenStack Architecture with HCI and Ceph
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/commit/aa63bf3931f74722dd48af8a0914233b2b384330) on Dec 19th, 2023**
```

Comment:
Please update this to indicate which commit of OpenStack operator you used to test this.

Reply:
I had a good run of this using the following. I could ask for Sergey to put that in the README with today's date. 

https://github.com/openstack-k8s-operators/openstack-operator/commit/ec7210b825a3c355007d1f1fc11a2952ba4a9262

Reply:
It has been added to latest update.